src/main/java/com/commercetools/sync/products/ProductSyncOptions.java

```diff
@@ -16,16 +16,17 @@
  * It is just provided as a skeleton/reminder for the options it should include in the future.
  */
 public class ProductSyncOptions extends BaseSyncOptions {
-    private boolean compareStaged = true; // to control whether to compare to the staged data or the published ones.
-    private boolean publish = false; // to control whether to auto-publish or not.
-    private boolean removeOtherVariants = true; // to control whether to remove other product variants or not.
+    private final boolean compareStaged; // to control whether to compare to the staged data or the published ones.
+    private final boolean updateStaged; // to control whether to update staged or current projection
+    private final boolean publish; // to control whether to auto-publish or not.
+    private final boolean removeOtherVariants; // to control whether to remove other product variants or not.
 
     // defines which attributes
-    private List<String> whiteList;
-    private List<String> blackList;
+    private final List<String> whiteList;
+    private final List<String> blackList;
```

Comment:
it's not currently handled but it will probably be just for convenience so that you can choose one of them. So if you wanted to only provide some attributes you can do that with whitelist instead of blacklisting all others and vice versa.